source/Nevermore/Advanced/Queryable/QueryTranslator.cs

```diff
@@ -183,11 +183,41 @@ protected override Expression VisitMethodCall(MethodCallExpression node)
                     sqlBuilder.Skip(skip);
                     return node;
                 }
+                case nameof(System.Linq.Queryable.Select):
+                {
+                    Visit(node.Arguments[0]);
+                    AddProjection(node.Arguments[1]);
+                    return node;
+                }
                 default:
                     throw new NotSupportedException();
             }
         }
 
+        void AddProjection(Expression expression)
```

Comment:
Disclaimer: A solid review of this requires a pretty deep understanding of what's going on. I feel like I'm about 75% there, but not all the way, so I'm mostly pattern-matching on code structure and sanity checking things; I'm highly unlikely to spot any actual bugs or correctness problems and I'm leaning on your unit tests to cover those.